packages/editor/package.json

```diff
@@ -142,6 +142,7 @@
         {
           "markupOnly": true,
           "ignoreAttribute": [
+            "eventKey",
```

Comment:
i'm pretty sure you don't need this anymore.

Reply:
Done